slider/beatmap.py

```diff
@@ -31,6 +31,145 @@ def _get(cs, ix, default=no_default):
         return default
 
 
+class Event:
+
+    EventType = Enum('EventType',
+                     {'Background': 0,
+                      'Video': 1,
+                      'Break': 2,
+                      'Sprite': 3,
+                      'Animation': 4})
+
+    @property
+    def start_time(self):
+        return self.start_time if self.start_time != 0 else None
+
+    @classmethod
+    def parse(cls, data):
+        event_type, start_time, *event_params = data.split(',')
+        try:
+            start_time = int(start_time)
+            start_time = timedelta(milliseconds=start_time)
```

Comment:
I'm assuming timedelta in this lib is used to refer to delta from beginning vs delta from last event?